master/internal/experiment.go

```diff
@@ -760,9 +760,6 @@ func (e *internalExperiment) handleContinueExperiment(reqID model.RequestID) (*i
 func (e *internalExperiment) processOperations(
 	ops []searcher.Operation, err error,
 ) {
-	if _, ok := model.StoppingStates[e.State]; ok {
-		return
-	}
```

Comment:
do you know why this is added in the first place?

I don't see much context on the PR https://github.com/determined-ai/pedl/pull/2907

To me I don't fully understand why this code is here, so it feels like a risky change to me

Reply:
I don't have the historical context, but I believe it's just saying that a CANCELLED experiment will trump status of failed experiments, if that happens during shut down. In the slack conversation linked in this JIRA ticket, I believe we're on agreement that this is a mistake. 
I forgot to add a release note for this fix -- once I do, I hope it will help you feel less risky about it.

Reply:
Are we sure this won't have any other unintended side effects? It seems like we call this from a lot of places within ```experiment.go``` and this function does a lot. 

This feels a like a chesterton's fence. Though I could be wrong I haven't dug into all usages yet.

Reply:
It was in the RM backlog, so I think it's a fair assumption that this is a ticket we wanted to work on. I think we should go ahead and merge this, give it 3 weeks to test on latest-ee. If it does have those unintended consequences, that can't be fixed, then I will happily revert this. Otherwise, I'd like to leave it in.

This change is only triggered on searcher stopping states, which is just a subset of searcher states. Additionally, I think there's a greater net positive of clearer logic for the devs/clients.

Reply:
amended the PR to only _not_ return when the searcher operation is "shutdown" -- this will help minimize unintended consequences